### ISSUE TITLE
Scarce information regarding IPFS-Conf in November

### ISSUE THREAD

**Issue by mib-kd743naq** [ipfs/conf#50]
Hi folks!

TL;DR: [This needs updating ( also why 2019 :thinking:)](https://github.com/ipfs/conf#ipfs-conf-lisbon-2019)

I will resume my whining about there being little information regarding an event that is supposed to take place in less than 3 months. Many of us have (BORING!) dayjobs and it would be rather difficult to negotiate a comfortable amount of time off on such short notice. Additionally many USians are already making turkey-day plans which once made are for all intents and purposes immutable ( see what I did there? ).

I personally want to come. And bring people with me. If I do not find out anything by the end of September - I almst certainly won't be able to :crying_cat_face: :panda_face: 

/cc @meiqimichelle @flyingzumwalt

**Comment by flyingzumwalt** [comment]
We have decided to postpone the IPFS Conf to 2019 in order to give ourselves more time to do a proper call for proposals, source the best presenters, and generally to make the event amazing. We are a bit behind on sending an email about this decision to the people who've pre-registered and we need to put up a blog post about it. 

In the meantime we've updated the existing blog post and the dev meeting repo to have accurate information.
- https://github.com/ipfs/blog/pull/173/files
- https://github.com/ipfs/conf/pull/51/files

The blog post and email will come soon. Meanwhile @mikeal has been working hard on assembling an events calendar for next year, while creating a Community Ops team and the IPFS ambassadors program, which will play big parts in IPFS Conf.

**Comment by meiqimichelle** [comment]
One more place to update -- this repo description. It says `Coordination for IPFS Conf 2018 and IPFS & libp2p Dev Meetings July 2018 https://conf.ipfs.io` . We may also want to remove the IPFS Conf url until there's something launched there :)

(I don't have permissions in this repo to edit myself, unfortunately.)

**Comment by mib-kd743naq** [comment]
@flyingzumwalt @meiqimichelle that's a bummer :crying_cat_face:  Here's to hoping 2019 will be a blast worth the wait! ( the IPFS-Dev-meet in Berlin was off-the-charts-amazing btw, I can't stress this enough! )

I will close this ticket for the time being
